recipes/simde/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "0.7.6":
+    url: "https://github.com/simd-everywhere/simde/archive/refs/tags/v0.7.6.tar.gz"
+    sha256: "c63e6c61392e324728da1c7e5de308cb31410908993a769594f5e21ff8de962b"
```

```suggestion
    url: "https://github.com/simd-everywhere/simde/releases/download/v0.7.6/simde-amalgamated-0.7.6.tar.xz"
    sha256: "703eac1f2af7de1f7e4aea2286130b98e1addcc0559426e78304c92e2b4eb5e1"
```

The release tar won't ever change its sha, something not guaranteed for archive urls (And something that GitHub in fact plans to do in the future as they already tried once not too long ago!)

If there's a release for that tag, it's usually better to use it

(The structure might be different, check locally  just in case :))

---

I suspected you would, yes, I wanted to make it a bit explicit so I have a message to link to next time I need to review the same thing :)

Thanks about the explanation on _why_ this one is better, much appreciated!